easybuild/easyconfigs/m/Multiwfn/Multiwfn-3.4.1-intel-2017b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CmdCp'
+
+name = 'Multiwfn'
+version = '3.4.1'
+
+homepage = 'http://sobereva.com/multiwfn/'
+description = "A multifunctional wavefunction analyzer"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['http://sobereva.com/multiwfn/misc/']
+sources = ['%(name)s_%(version)s_src_Linux.zip']
+checksums = ['07141333f59a4bb3e3855a0c6c2ac61ae0aa9009bd40b375848ea2b641d4ab8f']
+
+# The order of the source files is important!
+# The source files are copied from the original Makefile
```

Comment:
@smoors If there's a `Makefile`, why not use it? Maybe `MakeCp` is what you're after?